src/components/Header.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 
 const Header = () => (
-  <header>
+  <header className="font1">
     <h1>Library</h1>
     <nav>
       <Link to="/">BOOKS</Link>
```

Comment:
- [ ] The categories color is different from the design, kindly update it 
YOurs: ![image](https://user-images.githubusercontent.com/62705947/191987747-81528b96-3d83-4a03-9450-95eacb0bc13a.png)
Design: ![image](https://user-images.githubusercontent.com/62705947/191987996-1ab1640b-fbfd-4f75-8596-12a2b31a6c16.png)